src/pages-and-resources/xpert-unit-summary/settings-modal/SettingsModal.jsx

```diff
@@ -105,6 +109,72 @@ SettingsModalBase.defaultProps = {
   footer: null,
 };
 
+const ResetUnitsButton = ({
+  intl,
+  courseId,
+  checked,
+  visible,
+}) => {
+  const resetStatusRequestStatus = useSelector(getResetStatus);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (resetStatusRequestStatus === RequestStatus.SUCCESSFUL) {
+      setTimeout(() => {
+        dispatch(updateResetStatus({ status: '' }));
+      }, 2000);
```

Comment:
Why it takes 2 seconds for this reset update?

Reply:
To prevent user to spam the button and also to show the correct UI for all the states